news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Introducing a new bug severity classification"
+subtitle: "Discover how issues related to PrestaShop software are qualified and prioritized"
+date:   2019-05-29 11:30:00
+authors: [ MarionFrancois]
+icon: icon-code
+tags:
+ - qa
+---
+
+According to the [Information Technology Infrastructure Library (ITIL)](https://fr.wikipedia.org/wiki/Information_Technology_Infrastructure_Library), a bug is an "event that is not part of the standard operation of a service and causes an unplanned interruption or decrease the quality of service". And it is everybody’s job to chase them all!
```

Comment:
Looks like the quotes used are not standard : ’ is not '

Reply:
where ?

Reply:
Everywhere, but don't mind about that. Jekyll will transform quotes (`'`) to apostrophes (`’`) anyway when building the page.

Reply:
github preview is doing it too.